chia/cmds/stop.py

```diff
@@ -17,7 +17,10 @@ async def async_stop(root_path: Path, group: str, stop_daemon: bool) -> int:
     if stop_daemon:
         r = await daemon.exit()
         await daemon.close()
-        print(f"daemon: {r}")
+        if "data" in r and "success" in r["data"] and r["data"]["success"]:
```

Comment:
Maybe.

```suggestion
        if r.get("data", {}).get("success", False):
```

Reply:
Hm this doesn't fail if `success` is not in `r["data"]` right?